examples/time_evolution.jl

```diff
@@ -38,36 +36,36 @@ function main()
     # the number of circuit layers
     nl = 20
 
-    #The inds network
-    s = siteinds("S=1/2", g)
     # the initial state (all up, use Float 32 precision)
-    ψ0 = ITensorNetwork(ComplexF32, v -> "↑", s)
+    ψ0 = tensornetworkstate(ComplexF32, v -> "↑", g, "S=1/2")
```

Comment:
What do you think about defaulting `sitetype` to `"S=1/2"`?

Reply:
Yeah I like it. I'll do that.